src/main/sensors/gyro.h

```diff
@@ -58,6 +58,11 @@ typedef enum {
     GYRO_OVERFLOW_CHECK_ALL_AXES
 } gyroOverflowCheck_e;
 
+typedef enum {
+    BEFORE_STATIC_FILTERS = 0,
```

Comment:
These names are part of the global namespace, so they should be qualified: `DYN_FILTER_BEFORE_STATIC_FILTERS`,...

Also, unless they are used outside of `gyro.c`, they should not be made part of the interface, and should go into `gyro.c`.

Reply:
They are used outside of gyro.c, i.e. in gyroanalyse.c and gyro_filter_impl.h

I don't agree with pre-pending qualifiers.  We don't do it for other global namespace parameters like ON or OFF.  Likewise, antigravity has `SMOOTH` and `STEP` options, so the user reads: `set anti_gravity_mode SMOOTH`. 

To me, the shorter label reads more cleanly, i.e. `set dyn_filter_location BEFORE_STATIC_FILTERS'.  That's how I, as a user, would prefer to read it.  Can we please keep it like this?

Reply:
Wait, we are misunderstanding each other. The string that is used for CLI should stay as it is. The enum element should be qualified. This isn't exposed to users anywhere.

My bad for not seeing the use in `gyroanalyse.c`, you are correct on that. `gyro_filter_impl.h` (which should be `gyro_filter_impl.c` in my opinion, since it is not a header file) is a special case, as it has only internal scope to `gyro.c`, so this should not be treated as 'needs external scope'.

Reply:
Yep, sorry if that wasn't clear.

Reply:
No, it was me, as you know, I'm just not an experienced coder.